src/Configuration.php

```diff
@@ -96,6 +98,47 @@
     private const PHP_SCOPER_CONFIG = 'scoper.inc.php';
     private const DEFAULT_SIGNING_ALGORITHM = Phar::SHA1;
 
+    private const ALGORITHM_KEY = 'algorithm';
```

Comment:
I would suggest to put them into separated, non-instanceable class, or use prefix (instead of suffix) for easier distinguishability

Reply:
I still have mixed feelings about it as well to be honest, but for now I think I prefer to keep it as it is